lib/ui/notifications.dart

```diff
@@ -17,29 +18,43 @@ GlobalConfiguration cfg = GlobalConfiguration();
 
 class NotificationsWidgetState extends AssistantState<NotificationsWidget> {
   GlobalConfiguration cfg = GlobalConfiguration();
+  var notificationSubList = <NotificationSubscription>[];
+
+  var notificationSub = new NotificationSubscription('Test Keyword');  //TODO: create fetching new notification subscription
+
 
-  @override
   void initState() {
     super.initState();
   }
 
+  void addItemToList(){
+    setState(() {
+      notificationSubList.add(notificationSub);  //TODO: Add item to notification subscription list
```

Comment:
We can call `MailNotifier::addSubscription` here, and then refetch the subscription list.